dashboard/package.json

```diff
@@ -48,6 +48,7 @@
     "babel-preset-es2015": "~6.9.0",
     "babel-preset-react": "~6.5.0",
     "babel-preset-react-hmre": "~1.1.1",
+    "babel-register": "^6.22.0",
```

Comment:
Would be good to have a good default re: whether to use `^` or `~`. My $0.02 would be to be as conservative as possible...if we did strict equality I would be okay with that, but it looks like `~` is the most common thing.